src/Orc.CsvTextEditor/Extensions/StringExtensions.cs

```diff
@@ -127,6 +128,14 @@ public static string InsertCommaSeparatedColumn(this string text, int column, in
             return new string(textArray).TrimEnd(newLine);
         }
 
+        public static string RemoveEmptyLines(this string text)
+        {
```

Comment:
Either use `Argument.IsNotNull(() => text) or check if null, we don't want vague nullreferenceexceptions.